### PR TITLE
Update and fix integration test framework

### PR DESCRIPTION
**Pull request by nholland94**
- Made changes to automation configuration to support integration test framework again (will need to discuss with infra before landing).
  - moved back to `runtime_config` var instead of `genesis_ledger.json`; requires change to existing testnet configs
- Made changes to libp2p_helper to address connectivity issues.
  - half tested, seems promising, but requires some more iteration to confirm
  - probably needs chainsafe review

### CODE REVIEW

File: src/app/libp2p_helper/src/codanet.go [hunk 1]
```diff
@@ -243,6 +250,10 @@ func (gs *CodaGatingState) InterceptAccept(addrs network.ConnMultiaddrs) (allow
 		gs.logGate()
 	}
 
+	// if allow && gs.InternalAddrFilters.AddrBlocked(remoteAddr) {
```
Comment (imeckler) [inline src/app/libp2p_helper/src/codanet.go:253]: Delete?

**Review by imeckler — APPROVED** [review]
approved modulo removing commented out code

File: src/app/libp2p_helper/src/codanet.go [hunk 2]
```diff
@@ -257,7 +268,7 @@ func (gs *CodaGatingState) InterceptSecured(_ network.Direction, id peer.ID, add
 	// connections in coda are symmetric: if i am allowed to connect to
 	// you, you are allowed to connect to me.
 	remoteAddr := addrs.RemoteMultiaddr()
-	allow = gs.AllowedPeers.Contains(id) || (!gs.DeniedPeers.Contains(id) && !gs.AddrFilters.AddrBlocked(remoteAddr))
+	allow = gs.AllowedPeers.Contains(id) || (!gs.DeniedPeers.Contains(id) && !gs.AddrFilters.AddrBlocked(remoteAddr) && !gs.blockedInternalAddr(remoteAddr))
```
Comment (bkase) [inline src/app/libp2p_helper/src/codanet.go:271]: As discussed, please remove AddrFilters and blockedInternalAddr from here  -- blockedInternalAddr we don't want to check on incoming ever and AddrFilters we already check on InterceptAccept

**Review by bkase — APPROVED** [review]
Looks good minus that one change!

File: src/app/libp2p_helper/src/codanet.go (hunk 1 above)
Comment (nholland94) [reply src/app/libp2p_helper/src/codanet.go:253]: I'll remove this comment and open an issue instead.

**Review by 0x0I — APPROVED** [review]

**Pull request merged**